ta/pkcs11/src/sanitize_object.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2017-2020, Linaro Limited
+ */
+
+#ifndef PKCS11_TA_SANITIZE_OBJECT_H
+#define PKCS11_TA_SANITIZE_OBJECT_H
+
+#include "serializer.h"
+
+/*
+ * sanitize_consistent_class_and_type - Check object type matches object class
```

Comment:
I've got a pile of patches from which I've been asked to extract something that is reviewable and gives a somewhat sane history. If we make too many changes it will at some point make the extraction process impossible and I'll have to throw away what's left and start writing new code instead.

I can change the name of this particular function if you think that's important, but I'd like to keep the name of the file until this pile of patches is gone.

Reply:
Ok. Valid argument. Maybe just put a ticket in the backlog to refactor these names once everything is in and working.

Reply:
OK